### DISCUSSION TITLE
Full Page Livewire

### DISCUSSION THREAD

**Discussion by mrl22** [rappasoft/laravel-livewire-tables#325] [Q&A]
Hello

Great work on the package, I have been looking for something that works as simple as this for a while.

One question: Is it possible to load using full-page livewire from inside an existing live wire component?

I am trying to cut down on the number of components and blade files I am using.

Something like:

public function render()
    {
        return parent::render()
            ->extends('layouts.admin')
            ->section('content');
    }

Thank you.

**Comment by mrl22** [comment]
It seems that this code actually works :)

**Comment by lrljoe** [comment]
This will produce an error in v2+
